docs-src/go/how-to-trigger-a-schedule-in-go.md

```diff
@@ -0,0 +1,36 @@
+---
+id: how-to-trigger-a-schedule-in-go
+title: How to trigger a Schedule in Go
+sidebar_label: Trigger Schedule
+description: Immediately execute an Action in a Schedule.
+---
+
+Triggering a [Schedule](/concepts/what-is-a-schedule) immediately executes an [Action](/concepts/what-is-a-schedule#action) defined in that Schedule.
+By default, `trigger` is subject to the `AllowAll` Overlap Policy.
```

Comment:
I don't think that's true. Is that default coming from go sdk?

Reply:
Yes